apps/block_scout_web/lib/block_scout_web/templates/layout/app.html.eex

```diff
@@ -30,8 +30,8 @@
     <%= render_existing(@view_module, "styles.html", assigns) %>
 
     <link rel="apple-touch-icon" sizes="180x180" href="<%= static_path(@conn, "/apple-touch-icon.png") %>">
-    <link rel="icon" type="image/png" sizes="32x32" href="<%= static_path(@conn, "/images/favicon-32x32.png") %>">
-    <link rel="icon" type="image/png" sizes="16x16" href="<%= static_path(@conn, "/images/favicon-16x16.png") %>">
+    <link rel="icon" type="image/png" sizes="32x32" href="<%= static_path(@conn, "/images/favicon-32x32.png?v=M44lzPylqQ") %>">
```

Comment:
And the favicon icon files should be updated to use the EON logo: https://drive.google.com/drive/u/0/folders/17_KTGLTG5QfqkvtTnmNESfC3iUABuiQo

Reply:
New favicon images updated